label_studio/ml_models/models.py

```diff
@@ -75,3 +76,52 @@ class ThirdPartyModelVersion(ModelVersion):
 
     def has_permission(self, user):
         return user.active_organization == self.organization
+
+
+class ModelRun(models.Model):
+    class ProjectSubset(models.TextChoices):
+        ALL = 'All', _('All')
+        HASGT = 'HasGT', _('HasGT')
+
+    class FileType(models.TextChoices):
+        INPUT = 'Input', _('Input')
+        OUTPUT = 'Output', _('Output')
+
+    class ProjectStatus(models.TextChoices):
```

Comment:
Shouldnt this be called ModelRunStatus? And also I figure we want "In progress" or something

Reply:
yes sorry about that will fix it